notebooks/cognee_llama_index.ipynb

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
Should we add a test analogous to [test_notebook](https://github.com/topoteretes/cognee/blob/main/.github/workflows/test_notebook.yml) for this new notebook?

Reply:
@dexters1 , what do you think?

Reply:
We can, if we plan on having more notebooks in the future we can make my test_notebook action reusable as there will be little difference between notebooks

Reply:
I've added a test for this notebook in this MR, but the reusable action will be made in the next sprint. I don't have time to take care of it in this sprint.